browser/src/control/jsdialog/Widget.TreeView.ts

```diff
@@ -499,13 +499,24 @@ class TreeViewControl {
 			img.src = image;
 			img.alt = text;
 		} else {
-			const cell = L.DomUtil.create(
-				'span',
-				builder.options.cssClass +
-					` ui-treeview-cell-text ui-treeview-cell-text-content ui-treeview-${entry.row}-${index}`,
-				parent,
-			);
-			cell.innerText = text;
+			let cell;
+			if (treeViewData.id === 'searchfinds') {
```

Comment:
So this is not an optimal way of doing new features in the widget (hardcoding the id).
Imagine core will change id of the tree. Or we would like to reuse highlighting in another tree.

Best would be to inject into JSON some property. In this case I think we can assume `treeViewData.highlughtTerm !== undefined` is a marker that we want to highlight something.
Am I right?

That way we can use this nice feature in any tree and not be exposed for "changed id" bugs.